shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/EraMapping.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TypeFamilies #-}
+
+-- | This module provides the mapping from conceptual rule names to concrete
+-- rules for the Shelley era.
+module Shelley.Spec.Ledger.STS.EraMapping () where
```

Comment:
Minor: there's an inconsistency regarding the places where we define these instances. In ShelleyMA they are defined in: `module Cardano.Ledger.ShelleyMA.Rules.EraMapping`.